src/Sylius/Bundle/ShopBundle/Resources/views/Account/Order/Show/_payments.html.twig

```diff
@@ -9,7 +10,7 @@
                     {{ payment.method }}
                 </div>
                 <p>
-                    {{ payment.amount|sylius_price(payment.order.currencyCode) }}
+                    {{ money.convertAndFormat(payment.amount) }}
```

Comment:
Not sure if we should convert the value here, the customer will be charged in a base currency anyway /cc @pjedrzejewski